source/switch_payment_service_provider/switch_psp_to_stripe/index.html.md.erb

```diff
@@ -17,9 +17,14 @@ You must be a GOV.UK Pay admin user to switch PSP. To check if you are an admin
 
 ## Prepare to switch to Stripe
 
-Email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:mailto:govuk-pay-support@digital.cabinet-office.gov.uk) asking to switch your PSP to Stripe on all services for your organisation, or list out specific services.
+Email [govuk-pay-support@digital.cabinet-office.gov.uk](mailto:mailto:govuk-pay-support@digital.cabinet-office.gov.uk) asking to switch your PSP to Stripe. Your email must include:
 
-The GOV.UK Pay support team will confirm when we’ve enabled the switch PSP feature for your service.
+* a list of services you want to switch to Stripe
+* a clear payment descriptor of maximum 22 characters for each service - the payment descriptor is what will appear on your users' bank statements, not the service itself
```

Comment:
Can we also ask for statement descriptors that service wants on their bank statement ?